test/contrib/redshift_test.py

```diff
@@ -214,6 +216,123 @@ def test_s3_copy_to_temp_table(self, mock_redshift_target, mock_copy):
             (task.table,),
         )
 
+    @mock.patch("luigi.contrib.redshift.RedshiftTarget")
+    def test_s3_copy_with_valid_columns(self, mock_redshift_target):
+        task = DummyS3CopyToTableKey(columns=(
+            ('some_text', 'varchar(255)'),
+            ('some_int', 'int'),
```

Comment:
Since this is what is defined as the default, you should be able to just `task = DummyS3CopyToTableKey()`. Right?

Reply:
The idea was to emphasize on the valid columns types. But re-thinking it: I agree that it should be clear enough what the test is doing even when simply using the default `columns` attribute from `DummyS3CopyToTableKey()`.

Side note: For changes like this is there any preference whether to amend preceding commits or add them to a new commit?

Reply:
A new commit is fine. We squash/merge PRs and amend the message where necessary. So no need to rebase on your end (unless you just want to).